docs/source/markdown/options/device-read-bps.md

```diff
@@ -0,0 +1,6 @@
+#### **--device-read-bps**=*path*
+
+Limit read rate (bytes per second) from a device (e.g. --device-read-bps=/dev/sda:1mb).
```

Comment:
Please choose the version from `podman-run`, i.e., the one with 'path:rate' in the '####' line and asterisks in the example.